xm-commons-logging-web/src/main/java/com/icthh/xm/commons/logging/web/aop/RestLoggingAspect.java

```diff
@@ -159,11 +174,32 @@ private void logStart(final JoinPoint joinPoint, String method, final String[] c
             return;
         }
 
-        log.info("START {} : {} --> {}, input: {}",
-                 method,
-                 LogObjectPrinter.joinUrlPaths(controllerPath, methodPath),
-                 LogObjectPrinter.getCallMethod(joinPoint),
-                 LogObjectPrinter.printInputParams(joinPoint));
+        className = joinPoint.getSignature().getDeclaringType().getSimpleName();
```

Comment:
DO we need some conditions to allow Class, Package, and Method info extraction? Like the understanding, if we have logging.yml file or not?

Reply:
in current implementation of aspects inside LogObjectPrinter.getCallMethod class name extracted at every call, and we just move this call upper